dataflux_pytorch/benchmark/checkpointing/multinode/README.md

```diff
@@ -4,56 +4,115 @@ This benchmarking script will allow you to run and benchmark the performance of
 
 ## Getting started
 
-### Configuration
+### Setup
+ 
+#### Install requirements
+Create a Python virtual environment and activate it
+```shell
+python3 -m venv .venv
+source .venv/bin/activate
+```
+
+Run the following commands from the root of the repo to install the packages needed by the benchmarking code
+
+```shell
+pip install -r dataflux_pytorch/benchmark/requirements.txt`
```

Comment:
nit: looks like there's an extra back tick at the end of this line.

Reply:
Done.